packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+# Release Process for Base, Celocli, ContractKit, DAppKit, and Utils
+
+## Versioning
+
+Use the standard MAJOR.MINOR.PATCH semantic versioning scheme described at [semver.org](http://semver.org).
+
+New releases can be expected as follows:
+  * Major releases: approximately yearly 
+  * Minor releases: approximately 8 times a year
+  * Patch releases: as needed
+
+Development builds will be identified as such: `x.y.z-dev`, and will be published as `x.y.z` when stable. 
+
+## Identifying releases
+
+### NPM
+
+You can find the npm packages in the following places:
+  * [@celo/base](https://www.npmjs.com/package/@celo/base)
+  * [@celo/celocli](https://www.npmjs.com/package/@celo/celocli)
+  * [@celo/contractkit](https://www.npmjs.com/package/@celo/contractkit)
+  * [@celo/dappkit](https://www.npmjs.com/package/@celo/dappkit)
+  * [@celo/utils](https://www.npmjs.com/package/@celo/utils)
+
+### Github tags
+
+To identify the commits included in a specific release and see which new features were added or bugs fixed, please refer to the [release notes](https://github.com/celo-org/celo-monorepo/releases) in the monorepo. Also to keep track of continual updates to the stable and dev versions of the packages, each package has a `CHANGELOG.md` file: [Base](https://github.com/celo-org/celo-monorepo/blob/master/packages/base/CHANGELOG.md), [Celocli](https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/CHANGELOG.md), [Contractkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/CHANGELOG.md), [Dappkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/dappkit/CHANGELOG.md), [Utils](https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/CHANGELOG.md).
+All releases should be tagged with the version number, e.g. `contractkit-vX.Y.Z`. Each release should include a summary of the release contents, including links to pull requests and issues with detailed description of any notable changes. 
+
+### Communication
+
+The community will be notified of package updates through the following channels: 
+
+For all releases: 
+  * Each package’s `CHANGELOG.md` file, as mentioned above
+  * Github releases page, as mentioned above
+  * [Discord](https://discord.gg/6yWMkgM): #developer-chat, #mainnet, and #sdk
```

Comment:
Do you think it's worth putting Celo Forum as well? We need to do some more research into how much the forum is used and by which subsets of the comunity.

Reply:
Yup! the forum is already mentioned in this file, a few lines below this one :) Only for major releases, similar to [this blockchain release announcement](https://forum.celo.org/t/release-v1-1-0-of-celo-blockchain/616).